package.json

```diff
@@ -11,6 +11,8 @@
     "build:content": "node --max-old-space-size=8192 script/build-content.js",
     "build:content:test": "node script/test-cms-export.js",
     "build:webpack": "NODE_OPTIONS=--max-old-space-size=4096 webpack --config config/webpack.config.js",
+    "build:webpack:prod": "NODE_ENV=production yarn build:webpack --env.buildtype vagovprod",
+    "debug:webpack:prod": "NODE_ENV=production NODE_OPTIONS=--max-old-space-size=4096 node --inspect --inspect-brk ./node_modules/webpack/bin/webpack.js --config config/webpack.config.js --env.buildtype vagovprod",
```

Comment:
1. Run `debug:webpack:prod`
1. Open chrome://inspect
1. Click **Inspect** for the remote target (./node_modules/webpack/bin/webpack.js)

![debug-node](https://user-images.githubusercontent.com/6130520/94571153-463f2e00-0235-11eb-9c68-97fb6da33b48.gif)

Reference: https://webpack.js.org/contribute/debugging/